query/src/main/scala/filodb/query/exec/ExecPlan.scala

```diff
@@ -130,16 +130,18 @@ trait ExecPlan extends QueryCommand {
         .record(Math.max(0, System.currentTimeMillis - startExecute))
       span.mark(s"execute-step2-start-${getClass.getSimpleName}")
       FiloSchedulers.assertThreadName(QuerySchedName)
+      val transformerEmptySchema = allTransformers.filter(_.canHandleEmptySchemas)
```

Comment:
Nit: Rename as emptySchemaTransformers (optional)

Reply:
Please also add some english comments :) explaining the circus being done. It is non-obvious to new reader

Reply:
Sorry I missed the last comment. I will add in next PR